<a href="https://colab.research.google.com/github/ipark2/test_project/blob/main/6_10_Assignment_02.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [67]:
# 사전의 구조와 이해

price = {'김밥': 5000, '어묵':3000, '떡볶이':2000}

print(dict['김밥'], "\n",dic.values())

5000 
 dict_values([5000, 3000, 2000])


In [36]:
# 리스트의 슬라이싱을 사용하여 아래를 출력하세요

tp = [1,2,3,4,5,6,7]

print(tp[0], "\n",tp[:-4], "\n",tp[3::], "\n",tp[6], "\n",tp[::-1])


1 
 [1, 2, 3] 
 [4, 5, 6, 7] 
 7 
 [7, 6, 5, 4, 3, 2, 1]


In [70]:
# 조건문
# 점수를 입력받아 등급을 출력하세요
# 등급표 81~100	A / 61~80	B / 41~60	C / 21~40	D / 0~20	E

score = int(input("점수를 입력하세요: "))
if score >= 81 and score <= 100: 
    print("{}".format(score))
    print("점수는 {}점이며 등급은 A등급입니다".format(score))
elif score >= 61 and score <= 80:
    print("{}".format(score))
    print("점수는 {}점이며 등급은 B등급입니다".format(score))
elif score >= 41 and score <= 60:
    print("{}".format(score))
    print("점수는 {}점이며 등급은 C등급입니다".format(score))
elif score >= 21 and score <= 40:
    print("{}".format(score))
    print("점수는 {}점이며 등급은 D등급입니다".format(score))
elif score >= 0 and score <= 20:
    print("{}".format(score))
    print("점수는 {}점이며 등급은 F등급입니다".format(score))

#
######################


점수를 입력하세요: 85
85
점수는 85점이며 등급은 A등급입니다


In [ ]:
# 함수를 만들고 두 수를 입력받아 연산하세요

a = int(input('첫 번째 정수를 입력하세요: '))
b = int(input('두 번째 정수를 입력하세요: '))

# complete the blanks
#








#
######################


첫 번째 정수를 입력하세요: 3
두 번째 정수를 입력하세요: 5
덧셈 함수 연산:  8
곱셈 함수 연산:  15
제곱 함수 연산:  243


In [87]:
a = int(input('첫 번째 정수를 입력하세요: '))
b = int(input('두 번째 정수를 입력하세요: '))

def sum(a, b):
    v = sum(a,b)
    print(v)


첫 번째 정수를 입력하세요: 3
두 번째 정수를 입력하세요: 5


첫 번째 정수를 입력하세요: 3
두 번째 정수를 입력하세요: 5
덧셈 함수 연산:  8
곱셈 함수 연산:  15
제곱 함수 연산:  243


In [ ]:
# 알고리즘 생성
# 피보나치 수열을 계산하는 함수를 생성하고 자연수 10을 넣어 실행하세요

# complete the blanks
#



#
######################

print(pivo(10))


55


In [ ]:
# 클래스
# 사람 (Human) 클래스를 정의하고 (이름, 나이, 성별)을 받는 생성자를 추가하세요.
